eslint.config.mjs

```diff
@@ -46,30 +46,18 @@ const config = createConfig([
   },
   {
     rules: {
-      // Left disabled because various properties throughough this repo are snake_case because the
```

Comment:
Any rules that are removed from here are turned on by default. We are removing rules that are turned off that aren't being violated so they are caught in future. We still need to do more refinement here and will do so in a subsequent PR